packages/migrate/src/__tests__/DbPull.test.ts

```diff
@@ -1525,25 +1525,25 @@ describe('postgresql views introspection warnings', () => {
+                                                                                                                                                                                                                                                                                // The following views were ignored as they do not have a valid unique identifier or id. This is currently not supported by the Prisma Client. Please refer to the documentation on defining unique identifiers in views: https://pris.ly/d/view-identifiers
+                                                                                                                                                                                                                                                                                // - View "A"
+                                                                                                                                                                                                                                                                                // 
+                                                                                                                                                                                                                                                                                // These fields were commented out because their names are currently not supported by Prisma. Please provide valid ones that match [a-zA-Z][a-zA-Z0-9_]* using the \`@map\` attribute.
+                                                                                                                                                                                                                                                                                // - View "A", Field: "1"
+                                                                                                                                                                                                                                                                                // 
+                                                                                                                                                                                                            `)
       expect(ctx.mocked['process.stdout.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
       expect(ctx.mocked['process.stderr.write'].mock.calls.join('\n')).toMatchInlineSnapshot(``)
     })
   })
 })
 
-describe('postgresql partitioned tables introspection', () => {
+describe('postgresql introspection stopgaps', () => {
   const connectionString = process.env.TEST_POSTGRES_URI_MIGRATE!.replace('tests-migrate', 'tests-migrate-db-pull')
```

Comment:
Sure that it is ok to use the same connectionstring for all these tests? Not the danger that they cross over each other maybe?